src/thor/bidirectional_astar.cc

```diff
@@ -1016,6 +1040,10 @@ std::vector<std::vector<PathInfo>> BidirectionalAStar::FormPath(GraphReader& gra
       const BDEdgeLabel& edgelabel = edgelabels_forward_[edgelabel_index];
 
       const auto* edge = graphreader.directededge(edgelabel.edgeid(), tile);
+      if (edge == nullptr) {
+        throw std::runtime_error("BidirectionalAStar::FormPath can't find edge by id " +
```

Comment:
i think a more exact message would be in order here.

`"BidirectionalAStar::FormPath failed, tile no longer available " + std::to_string(edgelabel.edgeid().Tile_Base())`